### DISCUSSION TITLE
Перевод документации Tarantool на русский язык

### DISCUSSION THREAD

**Discussion by patiencedaur** [tarantool/doc#2738] [General]
## Ищем контрибьюторов-переводчиков

Хотите участвовать в переводе [документации Tarantool](https://www.tarantool.io/en/doc/latest/) на русский язык?

![State of localization](https://badges.crowdin.net/tarantool-docs/localized.svg)

- Мы используем [Sphinx](https://www.sphinx-doc.org/en/master/usage/advanced/intl.html#translating-with-sphinx-intl), а загрузка переводов на сайт автоматизирована с помощью GitHub Actions. [Техническая инструкция по локализации](https://www.tarantool.io/en/doc/latest/contributing/docs/build/#localization).
- Мы пользуемся [руководством по стилю переводов Tarantool](https://www.tarantool.io/en/doc/latest/contributing/docs/localization/guide/) и другими [рекомендациями для контрибьюторов Tarantool](https://www.tarantool.io/en/doc/latest/contributing/).

Вы можете предложить свою помощь в переводе готового документа или назначить на себя один из тикетов по тегу [translate](https://github.com/tarantool/doc/labels/translate) в этом репозитории.

Стало интересно? Оставьте комментарий ниже или свяжитесь с @veod32. Спасибо!

**Comment by snowlue** [comment]
👋🏻 Привет. Готов перевести #3146.

**Comment by KaymeKaydex** [comment]
Привет, готов перевести https://www.tarantool.io/ru/doc/latest/dev_guide/internals/box_protocol/

**Comment by AlexBikbaeva** [comment]
Привет, готова перевести https://www.tarantool.io/en/doc/latest/overview/
